content/applications/hr/payroll/work_entries.rst

```diff
@@ -80,95 +91,247 @@ there are any errors on the work entry, such as required fields being left blank
-:guilabel:`Name` field, typically listing the month and year for the particular batch.
+When the :guilabel:`Generate Payslips` button is clicked, a batch entry appears on a separate page
+for the time period selected.
 
-The date range the payslips apply to appears in the :guilabel:`Period` field. The company appears in
-the :guilabel:`Company` field, along with an option to mark the payslips as a credit note. To make
-changes, click the :guilabel:`Edit` button in the top left, make any changes, then click either
-:guilabel:`Save` to accept the changes or :guilabel:`Discard` to revert to the original data.
+The batch name populates the :guilabel:`Batch Name` field in a default `From (date) to (date)`
+format.
 
-.. image:: work_entries/batch.png
-   :align: center
-   :alt: Batch information that appears when making a batch.
+The date range to which the payslips apply appears in the :guilabel:`Period` field, and the company
+appears in the :guilabel:`Company` field. It is **not** possible to make changes to this form.
 
 Click the :guilabel:`Create Draft Entry` button to create the payslips for the batch.
 
-Click on the :guilabel:`Payslips` button in the top right to view all the payslips for the batch.
+Click the :guilabel:`Payslips` smart button at the top of the page to view all the payslips for the
+batch.
 
-.. note::
-   Payslips will have a status of *Waiting* until the :guilabel:`Create Draft Entry` button has been
-   clicked. After, the payslip status will change to *Done*.
+.. image:: work_entries/generate-payslips.png
+   :align: center
+   :alt: Information that appears when generating payslips.
+
+Printing payslips
+-----------------
+
+To print payslips, first view the individual payslips by clicking the :guilabel:`Payslips` smart
+button on the batch form.
+
+Next, select the payslips to print from the :guilabel:`Payslips` list. Click the box next to each
+payslip to print, or click the box to the left of the :guilabel:`Reference` column title, to select
+all the payslips in the list at once.
 
-Payslips can be printed by clicking the box next to each payslip to print, or clicking the box next
-to :guilabel:`Reference` to select all the payslips at once. Click the :guilabel:`Print` button, and
-a PDF file will be created with all the specified payslips.
+Click the :guilabel:`Print` button, and a PDF file is created with all the specified payslips.
 
 .. image:: work_entries/print-payslips.png
    :align: center
    :alt: Print button for printing the payslips.
 
+.. note::
+   The :guilabel:`Print` button does **not** appear until at least one payslip is selected in the
+   list.
+
+Time off to report
+==================
+
+If a time off request is submitted for a time period that was already processed on a payslip, the
+time off request appears in the :guilabel:`Time Off` page in the *Payroll* app, which is accessible
+by navigating to :menuselection:`Payroll app --> Work Entries --> Time Off to Report`.
```

Comment:
No need for a guilabel on the "_Time Off_ page" since that text is not on the active UI
```suggestion
time off request appears in the *Time Off* page in the *Payroll* app, which is accessible by
navigating to :menuselection:`Payroll app --> Work Entries --> Time Off to Report`.
```